lib/cellular/models/sms.rb

```diff
@@ -41,16 +40,6 @@ def delivered?
       @delivered
     end
 
-    def country
-      warn "[DEPRECATION] 'country' is deprecated; use 'country_code' instead"
-      @country_code
-    end
-
-    def country=(country)
-      warn "[DEPRECATION] 'country' is deprecated; use 'country_code' instead"
-      @country_code = country
-    end
-
```

Comment:
At least remove the specs that checks for the deprecation warnings and stuff. :smiley: